datadog/resource_datadog_synthetics_test_.go

```diff
@@ -259,9 +259,10 @@ func syntheticsTestRequest() *schema.Resource {
 
 func syntheticsTestRequestHeaders() *schema.Schema {
 	return &schema.Schema{
-		Description: "Header name and value map.",
-		Type:        schema.TypeMap,
-		Optional:    true,
+		Description:      "Header name and value map.",
```

Comment:
could you look into why the description of this field got removed from the generated docs? I guess it's something to do with the validate function.

Reply:
Need to use ValidateFunc instead of ValidateDiagFunc for some reason, although ValidateFunc is marked as deprecated https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#validatefunc